src/SourceDialog/SourceDialog.vala

```diff
@@ -246,13 +242,15 @@ public class Maya.View.SourceDialog : Gtk.Grid {
             type_combobox.sensitive = true;
             color_button_red.active = true;
             create_button.set_label (_("Create Calendar"));
+            is_default_check.active = false;
         } else {
             event_type = EventType.EDIT;
             create_button.set_label (_("Save"));
             name_entry.text = source.display_name;
             type_combobox.sensitive = false;
             type_combobox.set_active (0);
             list_store.foreach (tree_foreach);
+            is_default_check.active = false;
```

Comment:
Is this meant to be false on both sides of this conditional?

Reply:
No, it’s supposed to be this way. That’s what the last blurb in the main comment was about. It’s always unchecked, so I guess it’s something more like a “do I set this as default when the dialog exits“ check than a “default calendar” check. 

I don’t know of a good way to get the status of whether a calendar is default from here. Also, if I did make it reflect the default, then what would it mean to uncheck the current default calendar?

Reply:
Ah okay. Yeah I guess it should probably reflect the correct state on open and then if it's checked we could make it insensitive so you can't uncheck, only check

Reply:
Oh, that’s a good idea. I’ll see what I can do!

Reply:
Alright, here's my take at implementing that. It's much nicer this way!

Reply:
One design question I suppose. Should I change the label to something like "Calendar is already the default" to show why it's insensitive, or do you think this is enough?

Reply:
We can probably revisit that later if after some use it turns out it's confusing